steampipecloud/resource_steampipecloud_connection.go

```diff
@@ -319,17 +147,27 @@ func resourceConnectionRead(ctx context.Context, d *schema.ResourceData, meta in
 	client := meta.(*SteampipeClient)
 
 	// Warning or errors can be collected in a slice type
+	var connectionHandle, orgHandle string
 	var diags diag.Diagnostics
 	var err error
 	var r *http.Response
 	var resp steampipe.Connection
+	var isUser = false
+	id := d.Id()
+
+	ids := strings.Split(id, ":")
```

Comment:
What IDs are these? When do we get an ID like `part1:part2` and `part1`? A comment here would be helpful